app/views/drones/index.html.erb

```diff
@@ -4,3 +4,4 @@
 <div>
   <p><%= drone.name %></p>
```

Comment:
Just an idea.
On our cards (home index), we should display more info than just the drone.name.

For example we can add stars for reviews or the name of the owner.

More ideas are welcome of course!

Reply:
I totally agry. Will add this